randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1451,6 +1336,10 @@ Extra - polygon: [[-900.0, -7000.0], [-5000.0, -7000.0], [-5000.0, -10700.0], [-
       Gravity Suit and Morph Ball and After Burenia - Main Hub Bottom Grapple Block
   > Water Space Jump Platform
       Morph Ball and After Burenia - Main Hub Bottom Grapple Block
+  > Inside Tunnel
+      All of the following:
+          # https://youtu.be/yb_tJYnFFIw
+          Gravity Suit and After Burenia - Storm Missile Gate for Etank and Aim Down Clip (Advanced) and Can SSC
```

Comment:
Currently, door lock does not make this impossible. I specifically chose the event rather than no door lock because then you'd be expected to charge the spark from that room. This would only break once Flash Shift/Cloak doors can be shuffled.

Reply:
That's how i tried to handle the speedboost puzzle in hanubia via Tank Room before i realized it didnt actually matter. But before I realized it didnt matter, i realized it would be possible to activate the event but not be able to use that door.

Reply:
Oh I see the issue here now. This would become a huge problem for random start if you started in the Etank room, since it would require storms to leave.